app/models/person.rb

```diff
@@ -155,6 +155,18 @@ def self.search(query, user)
     ).includes(:profile).order(search_order)
   end
 
+  def self.contacts_search(query, user)
+    return where("1 = 0") if query.to_s.blank? || query.to_s.length < 2
```

Comment:
http://devdocs.io/rails~4.2/activerecord/querymethods#method-i-none

`query.blank?`, it's defined everywhere. Use `size` over `length`.